Libraries/ESA/esa.lisp

```diff
@@ -1008,10 +1008,6 @@ used.")
      (find-keystroke-item event table :errorp nil))))
 
 (defun set-key (command table gestures)
-  ;; WTF?
-  #-(and)
-  (unless (consp command)
-    (setf command (list command)))
```

Comment:
Should we replace this with `(check-type command cons)`?

Reply:
we do ensure-list in make-menu-item from now on, so this is not a problem.

I gather that the person who originally wrote the code consulted the spec and knew, that the command is a list and wanted to allow atoms, while next person did not, and assumed that this is bogus. First I've started to correct all places in McCLIM to pass lists (not symbols) where appropriate, however I've decided that it may break code which was already written, so if the command is atom, then we coerce it to a list in make-menu-item.